LayoutTests/fast/snapshot/positioned-static.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#box {
+    width: 100px;
+    height: 100px;
+    background-color: blue;
+    color: black;
+}
+</style>
+</head>
+<body>
+    <div id="box">Is this included?</div>
+    <canvas id="canvas"></canvas>
+<script>
+if (window.testRunner) {
+    testRunner.waitUntilDone();
+}
+
+async function main() {
+    const box = document.getElementById('box');
+    const canvas = document.getElementById('canvas');
+
+    if (!window.internals) {
+        console.log('FAIL: window.internals is not available');
+        return;
+    }
+
+    const imageData = internals.snapshotNode(box);
```

Comment:
Consider moving this shared JS into a `resources/snapshot-helper.js` file.

Reply:
done